source/acquaman/SGM/SGMMapScanConfiguration.cpp

```diff
@@ -0,0 +1,64 @@
+#include "SGMMapScanConfiguration.h"
+
+#include "acquaman/SGM/SGMMapScanController.h"
+#include "ui/SGM/SGMMapScanConfigurationView.h"
+#include "beamline/SGM/SGMBeamline.h"
+#include "beamline/SGM/SGMHexapod.h"
+
+SGMMapScanConfiguration::SGMMapScanConfiguration(QObject *parent)
+	: AMGenericContinuousScanConfiguration(parent)
+{
+	setName("SGM Map Scan");
+	setUserScanName("SGM Map Scan");
+
+	setControl(0, SGMBeamline::sgm()->hexapod()->xAxisPrimeControl()->toInfo());
+	scanAxes_.at(0)->regionAt(0)->setRegionStep(1);
+	setControl(1, SGMBeamline::sgm()->hexapod()->zAxisPrimeControl()->toInfo());
```

Comment:
I left it on purpose because whatever we implement will go there.

Sent from my iPhone

On Nov 26, 2015, at 3:25 PM, davidChevrier <notifications@github.com<mailto:notifications@github.com>> wrote:


In source/acquaman/SGM/SGMMapScanConfiguration.cpp<https://github.com/acquaman/acquaman/pull/1752#discussion_r46007345>:

> +#include "SGMMapScanConfiguration.h"
> +
> +#include "acquaman/SGM/SGMMapScanController.h"
> +#include "ui/SGM/SGMMapScanConfigurationView.h"
> +#include "beamline/SGM/SGMBeamline.h"
> +#include "beamline/SGM/SGMHexapod.h"
> +
> +SGMMapScanConfiguration::SGMMapScanConfiguration(QObject *parent)
> +     : AMGenericContinuousScanConfiguration(parent)
> +{
> +     setName("SGM Map Scan");
> +     setUserScanName("SGM Map Scan");
> +
> +     setControl(0, SGMBeamline::sgm()->hexapod()->xAxisPrimeControl()->toInfo());
> +     scanAxes_.at(0)->regionAt(0)->setRegionStep(1);
> +     setControl(1, SGMBeamline::sgm()->hexapod()->zAxisPrimeControl()->toInfo());


Given today's discussion, should this be yAxisPrimeControl()?

-
Reply to this email directly or view it on GitHub<https://github.com/acquaman/acquaman/pull/1752/files#r46007345>.